platform/ios/platform/darwin/test/MLNRendererConfigurationTests.m

```diff
@@ -135,13 +135,7 @@ - (void)testSettingMLNIdeographicFontFamilyNameWithPlistValue {
     // `MLNIdeographicFontFamilyName` set to an invalid font family names array value
     {
         NSString *localFontFamilyName = [config localFontFamilyNameWithInfoDictionaryObject:@[@"test font 1", @"test font 2", @"test font 3"]];
-        
-        NSString *systemFontFamilyName;
-#if TARGET_OS_IPHONE
-        systemFontFamilyName = [UIFont systemFontOfSize:0 weight:UIFontWeightRegular].familyName;
-#else
-        systemFontFamilyName = [NSFont systemFontOfSize:0 weight:NSFontWeightRegular].familyName;
-#endif
```

Comment:
If we want to fuse the platforms back together again, would we want to add this back for mac? Or just test with the local font?

Reply:
`systemFontFamilyName` does not get used in this block.